Nodejs/Product/Nodejs/Intellisense/IntellisenseController.cs

```diff
@@ -193,6 +193,13 @@ internal sealed class IntellisenseController : IIntellisenseController, IOleComm
                             UpdateCurrentParameter();
                         }
                         break;
+                    default:
+                        if (IsIdentifierFirstChar(ch) && _activeSession == null

```

Comment:
Yeah, the other isense code actually blocks that (reverseExpressionParser and VsAnalyzer.) otherwise Ctrl+space would trigger in comments etc.